etc/skate-style/palette.mss

```diff
@@ -48,15 +48,15 @@ Map { font-directory: url(./fonts); }
 @agriculture:       #F2E8B6;
 
 @building:          #f4f4f5;
-@building_line:     #ecedee;
+@building_line:     #DBDDE1;
```

Comment:
nit(non-blocking): "should" this be lowercase?...

Seems like a majority (but not all) of the other hex colors are in lowercase, seems like a reason to make it lowercase for "style"/"consistency"?

This is such a small thing I dont even know if it's worth mentioning lol.

(all the quotes are intended to convey that this suggestion really doesn't matter)

Here's a button to change it if you agree 😄 
```suggestion
@building_line:     #dbdde1;
```

Reply:
Yeah seems reasonable for consistency!